spec/helpers/application_helper/buttons/miq_ae_default_spec.rb

```diff
@@ -0,0 +1,59 @@
+describe ApplicationHelper::Button::MiqAeDefault do
+  let(:session) { Hash.new }
+  let(:view_context) { setup_view_context_with_sandbox({}) }
+
+  before do
+    Tenant.seed
+    feature_list = %w(
+      miq_ae_class_edit
+      miq_ae_domain_edit
+      miq_ae_class_copy
+      miq_ae_instance_copy
+      miq_ae_method_copy
+      miq_ae_namespace_edit
+    )
+    user = FactoryGirl.create(:user, :features => feature_list)
+    login_as user
+  end
+
+  before(:each) do
+    @domain = FactoryGirl.create(:miq_ae_domain)
```

Comment:
I guess we should use `lets` for this ones and similar cases, then you don't need to evaluate before each `it`.